x-pack/plugins/infra/public/pages/metrics/inventory_view/components/survey_kubernetes.tsx

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiButton, EuiFlexGroup, EuiFlexItem, EuiGlobalToastList } from '@elastic/eui';
+import React from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+import useLocalStorage from 'react-use/lib/useLocalStorage';
+import { useWaffleOptionsContext } from '../hooks/use_waffle_options';
+
+const KUBERNETES_TOAST_STORAGE_KEY = 'kubernetesToastKey';
+const KUBERNETES_FEEDBACK_LINK = 'https://ela.st/k8s-feedback';
+
+const openInNewTab = (url: string) => {
+  window.open(url, '_blank', 'noreferrer');
+};
+
+export const SurveyKubernetes = () => {
+  const { nodeType } = useWaffleOptionsContext();
+  const podNodeType: typeof nodeType = 'pod';
+
+  const [isToastSeen, setIsToastSeen] = useLocalStorage(KUBERNETES_TOAST_STORAGE_KEY, false);
+  const markToastAsSeen = () => setIsToastSeen(true);
+
+  return (
+    <>
+      {nodeType === podNodeType && (
+        <>
+          <EuiButton
+            href={KUBERNETES_FEEDBACK_LINK}
+            target="_blank"
+            color="warning"
+            iconType="editorComment"
+            data-test-subj="infra-kubernetes-feedback-link"
+          >
+            <FormattedMessage
+              id="xpack.infra.homePage.tellUsWhatYouThinkK8sLink"
+              defaultMessage="Tell us what you think! (K8s)"
+            />
+          </EuiButton>
+          {!isToastSeen && (
+            <EuiGlobalToastList
+              toastLifeTimeMs={Infinity}
+              dismissToast={() => {
+                markToastAsSeen();
+              }}
+              toasts={[
+                {
+                  id: 'k8s-toast',
+                  title: (
+                    <FormattedMessage
+                      id="xpack.infra.homePage.kubernetesToastTitle"
+                      defaultMessage="We need your help!"
+                    />
+                  ),
+                  color: 'primary',
+                  iconType: 'help',
+                  toastLifeTimeMs: 0x7fffffff, // Biggest possible lifetime because we control when it should be visible using isToastSeen
+                  text: (
+                    <>
+                      <p>
+                        <FormattedMessage
+                          id="xpack.infra.homePage.kubernetesToastText"
+                          defaultMessage="Help us design your Kubernetes experience by completing a feedback survey."
+                        />
+                      </p>
+                      <EuiFlexGroup justifyContent="flexEnd" gutterSize="s">
+                        <EuiFlexItem grow={false}>
+                          <EuiButton
+                            data-test-subj="infra-toast-kubernetes-survey-start"
+                            onClick={() => {
```

Comment:
**nit**: You could use the `href` property on this button to convert it to a link and still apply the onClick handler to mark the toast as seen, this way you won't need to recreate a function to open a new tab

Reply:
@tonyghiani I already tried that but the `EuiButton` component is not supporting both `onClick` and `href` at the same time: 
<img width="938" alt="image" src="https://user-images.githubusercontent.com/14139027/216963514-f6190e64-c60c-4a58-9a84-deab7f6be1b0.png">

Reply:
Mmm I see... I'd still prefer to use an anchor tag for opening an external window, but I can't recommend a better way to handle this if the component doesn't allow us to use both properties.
You can try to mute the linter and see if it behaves correctly with both, otherwise, feel free to keep the code you wrote at first.

Reply:
@tonyghiani I played with it and as a result, I decided to use the `onClickCapture` handler for this case - that way we have the same anchor link in the DOM as if we use `onClick`, the `openInNewTab` function is removed and only with the difference that it will be called during the event capture phase instead of bubbling phase. Also functionally it works fine in this case and there is no need to disable the linting of the file. Wdyt?

Reply:
LGTM, thanks for investing time in finding a solution!
**nit**: you could directly pass the markAsSeen reference instead of a new inline function

Reply:
Thanks, done [here](https://github.com/elastic/kibana/pull/150233/commits/0ddb8ee9666fe9a403273088a92d6644c13b738b) ✅